### DISCUSSION TITLE
Feature Request: Window Compare Presets

### DISCUSSION THREAD

**Discussion by kitkat0981** [OrcaSlicer/OrcaSlicer#6165] [Ideas]
Hi,
In making changes to my presets, it would be nice to have the ability to compare the current setting with another setting like for filament, it would be nice to see the differences between the current filament preset to the standard default filament preset or other presets.

I know that the Qidi Tech slicer has that feature which came from Prusa Slicer which also has that option.

See the following like :
https://help.prusa3d.com/article/compare-presets_301482

thanks and keep up the great work !

**Comment by Ergonomicmike** [comment]
Like [this](https://github.com/SoftFever/OrcaSlicer/issues/6080)?

**Comment by kitkat0981** [comment]
Oh damn, stupid me thanks

Regards,

Nelson
